vue-frontend/src/components/home/HomeHeader.vue

```diff
@@ -0,0 +1,160 @@
+<template>
```

Comment:
Let's arrange files in efficient way, we can create a `partials` folder and add all the header, footer files there, as the header and footer are common for a website, we can't say it's home page's header.